README.md

```diff
@@ -1,2 +1,9 @@
 # opg-modernising-lpa
-OPG Modernising LPA: Managed by opg-org-infra &amp; Terraform
+## Getting Started
+### Prerequisites
+* Docker and docker-compose
+
+### Installation
+```bash
+$ docker-compose up -d
```

Comment:
docker now supports a copmpose command. It's suprisingly easier to use than finding - :D 


```suggestion
$ docker compose up -d
```